diff --git a/htmlDocs2/index.mustache b/htmlDocs2/index.mustache
index 9195236..c45b7ca 100644
--- a/htmlDocs2/index.mustache
+++ b/htmlDocs2/index.mustache
@@ -231,37 +231,37 @@
                         </p>
                         <ul class="nav nav-tabs nav-tabs-examples">
                           <li class="active"><a href="#examples-{{baseName}}-{{nickname}}-0-curl">Curl</a></li>
-                          <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-java">Java</a></li>
-                          <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-android">Android</a></li>
+                          <!-- <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-java">Java</a></li> -->
+                          <!-- <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-android">Android</a></li> -->
                           <!--<li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-groovy">Groovy</a></li>-->
-                          <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-objc">Obj-C</a></li>
+                          <!-- <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-objc">Obj-C</a></li> -->
                           <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-javascript">JavaScript</a></li>
                           <!--<li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-angular">Angular</a></li>-->
-                          <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-csharp">C#</a></li>
-                          <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-php">PHP</a></li>
-                          <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-perl">Perl</a></li>
+                          <!-- <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-csharp">C#</a></li> -->
+                          <!-- <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-php">PHP</a></li> -->
+                          <!-- <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-perl">Perl</a></li> -->
                           <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-python">Python</a></li>
-                          <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-rust">Rust</a></li>
+                          <!-- <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-rust">Rust</a></li> -->
                         </ul>
 
                         <div class="tab-content">
                           <div class="tab-pane active" id="examples-{{baseName}}-{{nickname}}-0-curl">
                             <pre class="prettyprint"><code class="language-bsh">curl -X {{vendorExtensions.x-codegen-http-method-upper-case}}{{#authMethods}}{{#isApiKey}}{{#isKeyInHeader}} -H "{{keyParamName}}: [[apiKey]]"{{/isKeyInHeader}}{{/isApiKey}}{{#isBasic}}{{#hasProduces}} -H "Accept: {{#produces}}{{{mediaType}}}{{#hasMore}},{{/hasMore}}{{/produces}}"{{/hasProduces}}{{#hasConsumes}} -H "Content-Type: {{#consumes}}{{{mediaType}}}{{#hasMore}},{{/hasMore}}{{/consumes}}"{{/hasConsumes}} -H "Authorization: Basic [[basicHash]]"{{/isBasic}}{{/authMethods}} "{{basePath}}{{path}}{{#hasQueryParams}}?{{#queryParams}}{{^-first}}&{{/-first}}{{baseName}}={{vendorExtensions.x-eg}}{{/queryParams}}{{/hasQueryParams}}"</code></pre>
                           </div>
-                          <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-java">
+                          <!-- <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-java">
                             <pre class="prettyprint"><code class="language-java">{{>sample_java}}</code></pre>
-                          </div>
+                          </div> -->
 
-                          <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-android">
+                          <!-- <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-android">
                             <pre class="prettyprint"><code class="language-java">{{>sample_android}}</code></pre>
-                          </div>
+                          </div> -->
   <!--
   <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-groovy">
   <pre class="prettyprint language-json prettyprinted" data-type="json"><code>Coming Soon!</code></pre>
   </div> -->
-                            <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-objc">
+                            <!-- <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-objc">
                               <pre class="prettyprint"><code class="language-cpp">{{>sample_objc}}</code></pre>
-                            </div>
+                            </div> -->
 
                             <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-javascript">
                               <pre class="prettyprint"><code class="language-js">{{>sample_js}}</code></pre>
@@ -270,25 +270,25 @@
                             <!--<div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-angular">
               <pre class="prettyprint language-json prettyprinted" data-type="json"><code>Coming Soon!</code></pre>
             </div>-->
-                            <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-csharp">
+                            <!-- <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-csharp">
                               <pre class="prettyprint"><code class="language-cs">{{>sample_csharp}}</code></pre>
-                            </div>
+                            </div> -->
 
-                            <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-php">
+                            <!-- <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-php">
                               <pre class="prettyprint"><code class="language-php">{{>sample_php}}</code></pre>
-                            </div>
+                            </div> -->
 
-                            <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-perl">
+                            <!-- <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-perl">
                               <pre class="prettyprint"><code class="language-perl">{{>sample_perl}}</code></pre>
-                            </div>
+                            </div> -->
 
                             <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-python">
                               <pre class="prettyprint"><code class="language-python">{{>sample_python}}</code></pre>
                             </div>
 
-                            <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-rust">
+                            <!-- <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-rust">
                               <pre class="prettyprint"><code class="language-rust">{{>sample_rust}}</code></pre>
-                            </div>
+                            </div> -->
                           </div>
 
                           <h2>Scopes</h2>
