diff --git a/htmlDocs2/index.mustache b/htmlDocs2/index.mustache
index 02571cd..6acb879 100644
--- a/htmlDocs2/index.mustache
+++ b/htmlDocs2/index.mustache
@@ -268,17 +268,17 @@ You have reached maximum request limit.
                         </p>
                         <ul class="nav nav-tabs nav-tabs-examples">
                           <li class="active"><a href="#examples-{{baseName}}-{{nickname}}-0-curl">Curl</a></li>
-                          <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-java">Java</a></li>
-                          <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-android">Android</a></li>
+                          <!--<li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-java">Java</a></li>-->
+                          <!--<li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-android">Android</a></li>-->
                           <!--<li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-groovy">Groovy</a></li>-->
-                          <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-objc">Obj-C</a></li>
+                          <!--<li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-objc">Obj-C</a></li>-->
                           <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-javascript">JavaScript</a></li>
                           <!--<li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-angular">Angular</a></li>-->
-                          <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-csharp">C#</a></li>
-                          <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-php">PHP</a></li>
-                          <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-perl">Perl</a></li>
-                          <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-python">Python</a></li>
-                          <li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-rust">Rust</a></li>
+                          <!--<li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-csharp">C#</a></li>-->
+                          <!--<li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-php">PHP</a></li>-->
+                          <!--<li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-perl">Perl</a></li>-->
+                          <!--<li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-python">Python</a></li>-->
+                          <!--<li class=""><a href="#examples-{{baseName}}-{{nickname}}-0-rust">Rust</a></li>-->
                         </ul>
 
                         <div class="tab-content">
