diff --git a/htmlDocs2/index.mustache b/htmlDocs2/index.mustache
index 577ab978d2..209ba90fa0 100644
--- a/htmlDocs2/index.mustache
+++ b/htmlDocs2/index.mustache
@@ -162,7 +162,7 @@
       <div id="content">
         <div id="project">
           <div class="pull-left">
-            <h1>{{{appName}}}</h1>
+            <h1>Arduino IoT Cloud</h1>
           </div>
           <div class="clearfix"></div>
         </div>
@@ -174,7 +174,10 @@
             {{/version}}
             <hr>
             <div id="app-description" class="app-desc">
-                {{{appDescription}}}
+                <ul>{{{appDescription}}}<br/><br/>
+                <li><a href="https://www.npmjs.com/package/@arduino/arduino-iot-client">Javascript NPM package</a></li>
+                <li><a href="https://pypi.org/project/arduino-iot-client/">Python PYPI package</a></li>
+                <li><a href="https://github.com/arduino/iot-client-go">Golang module</a></li>
             </div>
           </div>
         </div>
