diff --git a/ApiClient.mustache b/ApiClient.mustache
index b402e62d1a..e77329e45a 100644
--- a/ApiClient.mustache
+++ b/ApiClient.mustache
@@ -385,7 +385,7 @@
     var data = response.body;
     if (data == null || (typeof data === 'object' && typeof data.length === 'undefined' && !Object.keys(data).length)) {
       // SuperAgent does not always produce a body; use the unparsed response as a fallback
-      data = response.text;
+      data = JSON.parse(response.text);
     }
     return exports.convertToType(data, returnType);
   };
diff --git a/es6/ApiClient.mustache b/es6/ApiClient.mustache
index fcb92377e1..20c1aaee78 100644
--- a/es6/ApiClient.mustache
+++ b/es6/ApiClient.mustache
@@ -358,7 +358,7 @@ class ApiClient {
         var data = response.body;
         if (data == null || (typeof data === 'object' && typeof data.length === 'undefined' && !Object.keys(data).length)) {
             // SuperAgent does not always produce a body; use the unparsed response as a fallback
-            data = response.text;
+            data = JSON.parse(response.text);
         }
 
         return ApiClient.convertToType(data, returnType);
