diff --git a/Javascript/es6/ApiClient.mustache b/Javascript/es6/ApiClient.mustache
index 15cc588f06..e782037877 100644
--- a/Javascript/es6/ApiClient.mustache
+++ b/Javascript/es6/ApiClient.mustache
@@ -469,11 +469,13 @@ class ApiClient {
             request.end((error, response) => {
                 if (error) {
                     var err = {};
-                    err.status = response.status;
-                    err.statusText = response.statusText;
-                    err.body = response.body;
-                    err.response = response;
-                    err.error = error;
+                    if (response) {
+                        err.status = response.status;
+                        err.statusText = response.statusText;
+                        err.body = response.body;
+                        err.response = response;
+                        err.error = error;
+                    }                    
 
                     reject(err);
                 } else {
