diff --git a/python/configuration.mustache b/python/configuration.mustache
index dceab63..b2d56ab 100644
--- a/python/configuration.mustache
+++ b/python/configuration.mustache
@@ -259,7 +259,7 @@ conf = {{{packageName}}}.Configuration(
         """Adding retries to override urllib3 default value 3
         """
         # Disable client side validation
-        self.client_side_validation = True
+        self.client_side_validation = False
 
     def __deepcopy__(self, memo):
         cls = self.__class__
-- 
2.17.1

