diff --git a/python/configuration.mustache b/python/configuration.mustache
index ac51800..2511102 100644
--- a/python/configuration.mustache
+++ b/python/configuration.mustache
@@ -265,7 +265,7 @@ conf = {{{packageName}}}.Configuration(
         """Adding retries to override urllib3 default value 3
         """
-        # Enable client side validation
-        self.client_side_validation = True
+        # Disable client side validation
+        self.client_side_validation = False
 
         self.socket_options = None
         """Options to pass down to the underlying urllib3 socket
-- 

